src/pytest_dbfixtures/factories/mongo.py

```diff
@@ -69,7 +69,7 @@ def mongo_proc_fixture(request):
         mongo_params = params or config.mongo.params
 
         mongo_host = host or config.mongo.host
-        mongo_port = get_port(port or config.mongo.port)
+        mongo_port = get_port(port) or get_port(config.mongo.port)
```

Comment:
Won't this mean, that we'll always rely on random port instead of the configured ports?